README.md

```diff
@@ -105,18 +176,31 @@ go build
 ./dexc --testnet
 ```
 
-This will open up a terminal-based interface for the client.
+Connect to the client from your browser at `localhost:5758`.
+
+## Contribute
 
-## Development
+**Looking to contribute? We need your help** to make DEX &#35;1.
 
-Before starting work, chat with us in the
+Nearly all development is done in Golang and JavaScript. Work is coordinated
+through [the repo issues](../../issues), so that's the best place to start.
```

Comment:
Why not put the full URL here so that it opens to the right place no matter where the md is being viewed.